README.adoc

```diff
@@ -2297,7 +2297,7 @@ Prefer the use of exceptions from the standard library over introducing new exce
 === Parallel Assignment [[parallel-assignment]]
 
 Avoid the use of parallel assignment for defining variables.
-Parallel assignment is allowed when it is the return of a method call, used with the splat operator, or when used to swap variable assignment.
+Parallel assignment is allowed when it is the return of a method call, used with the splat operator, or when used to swap variable assignment, or `Hash#values_at` and `Hash#fetch_values`.
```

Comment:
These built-in methods are very convenient, but there are many such methods.

Reply:
Those are method calls, so it's already covered in this sentence.